models/skipthoughts.py

```diff
@@ -0,0 +1,194 @@
+"""
+A simple model based on skipthoughts sentence embeddings.
+
+To set up:
```

Comment:
Seems like a lot of this is covered again and better in pysts.embedding.SkipThoughts - I'd suggest just moving taht here.